rpc/jsonrpc/types/walletsvrcmds_test.go

```diff
@@ -973,6 +973,26 @@ func TestWalletSvrCmds(t *testing.T) {
 				CommentTo: dcrjson.String("commentto"),
 			},
 		},
+		{
+			name: "sendtoaddress optional2",
+			newCmd: func() (interface{}, error) {
+				return dcrjson.NewCmd("sendtoaddress", "1Address", 0.5, "comment", "commentto", dcrjson.Bool(true))
```

Comment:
This reads strangely as Bitcoin, not Decred, addresses begin with 1.

Reply:
right, all of the address-related tests in this file seem to use "1Address" as well. will clean up the strings.